docs/platform/howto/vpc-peering-gcp.rst

```diff
@@ -0,0 +1,30 @@
+.. _vpc-peering-gcp:
+
+Set Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)
+======================================================================
+
+Once you've created a :ref:`VPC on the Aiven platform <platform_howto_setup_vpc_peering>`, you can follow this instruction to set up VPC peering on AWS.
+
+1. Open your GCP Console.
+
+2. Select **VPC Networks** on the left and find the VPC that you want to connect.
```

Comment:
```suggestion
2. Select **VPC Networks** and find the VPC that you want to connect to.
```

Reply:
I would suggest removing "on the left" from here, if reasonable, to avoid needing to check/update things later if GCP changes their console UI.